frontend/common/src/components/Pagination/Pagination.tsx

```diff
@@ -231,6 +231,11 @@ const Pagination: React.FunctionComponent<PaginationProps> = ({
                 <select
                   style={{ cursor: "pointer" }}
                   value={pageSize}
+                  aria-label={intl.formatMessage({
+                    defaultMessage: "Change page size",
```

Comment:
Just because we don't use verbs on any other field, and shorter seems preferable for a11y 🤷 

```suggestion
                    defaultMessage: "Page size",
```